probe-rs/src/architecture/riscv/registers.rs

```diff
@@ -1,223 +1,235 @@
 use crate::{
-    core::{CoreRegister, RegisterDataType, RegisterId, RegisterRole},
+    core::{CoreRegister, RegisterDataType, RegisterId, RegisterRole, UnwindRule},
     CoreRegisters,
 };
 use once_cell::sync::Lazy;
 
 /// The program counter register.
 pub const PC: CoreRegister = CoreRegister {
-    name: "pc",
-    roles: &[RegisterRole::ProgramCounter],
+    roles: &[RegisterRole::Core("pc"), RegisterRole::ProgramCounter],
```

Comment:
We should decide if register names should be capitalized or not :)

Reply:
I went with what architecture docs use. ARM seems to capitalize most (not all), and RISCV seems to use lower case.

Where there are things that are cross-architecture relevant, I created a common role, like `RegisterRole::ProgramCounter`, to avoid confusion.

Reply:
Ok, that's fair, so we have the role and the name :) Good idea =)